python_modules/dagster-externals/dagster_externals/_context.py

```diff
@@ -1,10 +1,20 @@
 import atexit
+from contextlib import ExitStack
 from typing import Any, ClassVar, Mapping, Optional, Sequence
 
 from typing_extensions import Self
 
-from ._io.base import ExternalExecutionContextLoader, ExternalExecutionMessageWriter
-from ._io.default import ExternalExecutionFileContextLoader, ExternalExecutionFileMessageWriter
+from ._io.base import (
+    ExternalExecutionContextLoader,
+    ExternalExecutionMessageChannel,
```

Comment:
I think we should consider "WriterChannel" to make it clear that this is very specific to the Writer class. Could even make it an inner class

Reply:
I'm good w/ rename, especially if we are going to be shortening the `ExternalExecution` prefix. Prefer to avoid inner class for now just because it complicates things.